skopt/gp_opt.py

```diff
@@ -94,19 +94,17 @@ def gp_minimize(func, bounds=None, search="sampling", random_state=None,
 
     Returns
     -------
-    x_val: array-like
-        Parameter value corresponding to the best function value.
-
-    func_val: float
-        Function minimum.
-
-    d: dict
-        d["models"]: List of GP models of len maxiter. d["models"][i]
-        corresponds to the GP model fit in iteration i.
-        d["x_iters"]: d["x_iters"][i] corresponds to the "best" x in
-        iteration i.
-        d["func_vals]: d["func_vals"][i] corresponds to the minimum
-        function value in iteration i.
+    res: OptimizeResult
```

Comment:
I'm not sure this is necessary right now, since `n_iter > maxiter` is the only reason it can stop